### ISSUE TITLE
`yggdrasilctl --help` is missing multiple options and doesn't tell me that I want `yggdrasilctl help`instead

### ISSUE THREAD

**Issue by Mikaela** [yggdrasil-network/yggdrasil-go#194]
I am using 0.2.7 (`yggdrasil --version` doesn't exist as a side note) on Debian Testing.

* * * * *

https://yggdrasil-network.github.io/admin.html has at least getSessions and seems to include a longer list than:

```
Usage of yggdrasilctl:                                                         
  -endpoint string                                                             
        Admin socket endpoint (default "tcp://localhost:9001")                 
  -json                                                                        
        Output in JSON format     
```

I also thought that I saw getPeers in the help text.

* * * * *

```
                                  fields                                       
addPeer                           [uri [interface]]                            
getSelf                           []                                           
getSwitchPeers                    []                                           
getSwitchQueues                   []                                           
removePeer                        [port]                                       
addAllowedEncryptionPublicKey     [key]                                        
getAllowedEncryptionPublicKeys    []                                           
dot                               []                                           
getMulticastInterfaces            []                                           
getTunTap                         []                                           
help                              <nil>                                        
removeAllowedEncryptionPublicKey  [key]                                        
getDHT                            []                                           
getPeers                          []                                           
getSessions                       []                                           
setTunTap                         [name [tap_mode] [mtu]]                     
```

**Comment by neilalexander** [comment]
In `develop`, the commands are now under `yggdrasilctl list` instead of `yggdrasilctl help` since 315aadae067be7d68e07c2e15628313a20db3f7f.

We may need to improve the `--help` text somehow - need to think on that one, since it is generated by Go's `flag` module.

**Comment by Mikaela** [comment]
I think `help` should mention that it's deprecated

```
└┌(%:~)┌- yggdrasilctl --help
Usage of yggdrasilctl:
  -endpoint string
        Admin socket endpoint (default "tcp://localhost:9001")
  -json
        Output in JSON format
  -v    Verbose output (includes public keys)
└┌(%:~)┌- yggdrasilctl help  
Unspecified error occured
└┌(%:~)┌- yggdrasilctl --version
flag provided but not defined: -version
Usage of yggdrasilctl:
  -endpoint string
    	Admin socket endpoint (default "tcp://localhost:9001")
  -json
    	Output in JSON format
  -v	Verbose output (includes public keys)

**Comment by neilalexander** [comment]
This should be pretty much fixed with #232. Please close if you don't have any other problems with the below sample output.

```
yggdrasilctl -help
Usage: ./yggdrasilctl [options] command [key=value] [key=value] ...
Options:
  -endpoint string
    	Admin socket endpoint (default "tcp://localhost:9001")
  -json
    	Output in JSON format (as opposed to pretty-print)
  -v	Verbose output (includes public keys)
Commands:
  - Use "list" for a list of available commands
Examples:
  -  ./yggdrasilctl list
  -  ./yggdrasilctl getPeers
  -  ./yggdrasilctl setTunTap name=auto mtu=1500 tap_mode=false
  -  ./yggdrasilctl -endpoint=tcp://localhost:9001 getDHT
  -  ./yggdrasilctl -endpoint=unix:///var/run/ygg.sock getDHT
```

**Comment by Mikaela** [comment]
I think it could use some empty lines for readability between the topics, and I am not sure whether "Commands" is too "hidden" there, but I think it works for me.
